lib/drivers/node-mongodb-native/collection.js

```diff
@@ -77,11 +77,16 @@ function iter(i) {
     const collection = this.collection;
     const args = Array.from(arguments);
     const _this = this;
-    const debug = _this &&
+    const hasGlobalDebug = _this &&
```

Comment:
@vkarpov15 Can we use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here or should it be done like this for backwards compatibility?

Reply:
For backwards compatibility. Optional chaining is supported natively in Node 14 and higher, but Mongoose 6 supports Node 12.